index.html

```diff
@@ -1,14 +1,259 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="\styles\style.css" rel="stylesheet" />
+</head>
+
+<body>
+  <section id="header">
+    <header>
+      <a href="/index.html">
+        <h1>HOME PAGE</h1>
+      </a>
+      <h6>The page for the home</h6>
+    </header>

```

Comment:
These headers look like they are styled this was intentionally, but something about them seems off to me. Adding a little bit of space between the header and the sub header might make it a little easier on the user